community_images/common/helpers.sh

```diff
@@ -285,16 +285,6 @@ function report_pulls()
 }
 
 function finish {
-    if [[ -z "$NAMESPACE_TO_CLEANUP" ]]; then
```

Comment:
why are we removing this?

Reply:
I guess there was some issue with this part before. Checked it again, it worked without removing this part. Added it again. Thanks.